Google.Api.Generator/Generation/SnippetCodeGenerator.cs

```diff
@@ -155,7 +191,41 @@ private class SnippetDef
                 MethodGenerator(ctx, true)
                     .WithXmlDoc(XmlDoc.Summary($"Snippet for {TargetMethodName}"));
 
-            public NamespaceDeclarationSyntax StandaloneSnippet(SourceFileContext ctx)
+            private SnippetMetadata GenerateMetadata() =>
+                new SnippetMetadata
+                {
+                    Title = SnippetMethodName,
+                    Description = TargetMethod.DocLines.FirstOrDefault() ?? $"Snippet for {TargetMethodName}",
```

Comment:
I'm not sure this will give nice results. The first line of the method docs may not even be a complete sentence.
I'd be interested to see what this looks like for real APIs. Although maybe this is the generated doc lines that say "Helper for calling Xyz" or whatever? Again, I want to see what it actually does :) (That may well be in the tests later...)

Reply:
I would personally just go with $"Snippet for {TargetMethodName}" - I suspect that will actually be more meaningful.

Reply:
Done, on the second to last commit.